resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/KubernetesVolumeUtilsSuite.scala

```diff
@@ -117,4 +117,61 @@ class KubernetesVolumeUtilsSuite extends SparkFunSuite {
     }
     assert(e.getMessage.contains("hostPath.volumeName.options.path"))
   }
+
+  test("Parses nfs volumes correctly") {
+    val sparkConf = new SparkConf(false)
+    sparkConf.set("test.nfs.volumeName.mount.path", "/path")
+    sparkConf.set("test.nfs.volumeName.mount.readOnly", "true")
+    sparkConf.set("test.nfs.volumeName.options.path", "/share")
```

Comment:
In this case, we don't need `sparkConf.set("test.nfs.volumeName.options.readOnly", "true")`?

Reply:
I was checking the mount.readOnly option, aside from options.readOnly option. It is removed now.